src/client/Microsoft.Identity.Client.Broker/BrokerExtension.cs

```diff
@@ -23,6 +25,25 @@ public static class BrokerExtension
         /// If a broker does not exist or cannot be used, MSAL will fallback to a browser.
         /// Make sure browser auth is enabled (e.g. if using system browser, register the "http://localhost" redirect URI, etc.)
         /// </remarks>
+        public static PublicClientApplicationBuilder WithWindowsBroker(this PublicClientApplicationBuilder builder, bool enableBroker = true)
```

Comment:
Why do we still have this API? I thought we settled on the WithBroker(Options)  API?

Reply:
This was used by pure net6.0 (not net6.0-windows) apps. If this is removed and moved to WithBroker, only WamAdapter will be left in the package. We can discuss.

Reply:
Removed.